### PR TITLE
tyk/master [SYSE-15] Release branch creation has changed

### PR DESCRIPTION
**Pull request by alephnull**
The complete list of features for _tyk:master_: releng.

:robot: Beep, boop, this PR has been generated by [automation](https://github.com/TykTechnologies/gromit/blob/master/cmd/policy.go). Changes should be limited to templated code. Where allowed, this PR will be automatically merged when the branch protection rules are satisfied. If you think this was made by mistake, or if merging this might cause any instabilities, please bring it up on Slack or log a jira in the TD project.

### CODE REVIEW

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit:  
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6649635667)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: 6557816f046c833d2165af93d6a3e6f737e9eb34 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6649765231)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: f30e5f7e524605f0f5bc8f0a7cdd8431ce62b021 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6649777393)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: 901381eab8ebd161cefbbf2868cc8dbf0c5add1b 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6649875579)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: 29e50708788c10acdd7b6282be252d4eae7b5a7a 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6649969411)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: 20d3ef1582042fbbfe4b911a861aae3ea5ec21ee 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6649976784)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: 4e225ad2fc16fa5d92ed121fab3b73479ea25edd 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6652172322)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: e92034b675fc9b5e64a5d815ab3ba5832e92c8b8 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6652553558)

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -35,12 +33,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        golang_cross: [ 1.19-bullseye ]
+        golang_cross:
+          - 1.19-bookworm
+        golang_cross: [ 1.19-bookworm ]
```
Comment (asutosh) [inline .github/workflows/release.yml:38]:
```
        golang_cross:
          - 1.19-bookworm
        golang_cross: [ 1.19-bookworm ] 
```
There is duplication in golang_cross

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: bc2a8a8f7afa4b5b9f79ab4e1fd0ec03b9edebfd 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6655340366)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: 1b0928397bdfe29952a2b4f0a5edbdbab9275307 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6656110460)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: ac2833c53f2e2643ed41861b77c728d51c00276a 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6657686603)

**Comment by buger** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/5681/merge
Commit: 5b87ea18229b65030dfc0b505d72a57a971d471f 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6662122771)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: 5b2f987938aeb27671f238229e787faeae9fef6b 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6662251132)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: d59ddc0d7c7eaaba9161836df71460db55534df3 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6662326342)

**Comment by buger** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/5681/merge
Commit: 241ae3c765ac532d98829b483bffc2197e727e16 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6662529423)

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: 8f2a2ff185fef3ef9ff1da5cb4cf5cefe91e74b8 
Triggered by: pull_request (@alephnull)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6662757296)

**Review by asutosh — APPROVED** [review]

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5681/merge
Commit: 96b29a006308a5c58e492ee0b7041caf9a4a31c8 
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6662934754)

**Pull request merged**